pkg/kubectl/cmd/auth/cani.go

```diff
@@ -193,7 +193,10 @@ func (o *CanIOptions) Validate() error {
 		}
 		return nil
 	}
-
+	err := isVerb(o.Verb)
```

Comment:
do distinct checks depending on whether we're checking resource or nonResourceURL permissions. warn, but continue on unknown verbs